DistributedCollective/Sovryn-frontend
# Origin FISH vesting withdraw

Testing:
1. open portfolio page -> vested assets
2. verify "Origin FISH" row is visible
3. verify "Origin FISH" has "Withdraw" button
4. Withdraw your unlocked balance if any, verify tx confirms.

"Origin FISH" row will not appear if your wallet has no FISH vestings (didn't participate in testnet sale)
